homeassistant/components/vacuum/neato.py

```diff
@@ -93,14 +93,17 @@ def update(self):
             else:
                 self._clean_state = STATE_IDLE
                 self._status_state = 'Stopped'
+
+            if ALERTS.get(self._state['alert']) is not None:
```

Comment:
Since we want to use the value of the item below, save the value in a variable first, then check it, then set it if the check passes.